egs/wsj/s5/steps/nnet3/components.py

```diff
@@ -88,6 +88,17 @@ def AddLdaLayer(config_lines, name, input, lda_file):
     return {'descriptor':  '{0}_lda'.format(name),
             'dimension': input['dimension']}
 
+def AddFixedAffineLayer(config_lines, name, input, lda_file):
```

Comment:
rename the last argument to parameter file. Could you also modify AddLdaLayer to call AddFixedAffineLayer.